client/src/lib/daily/DailyProvider.tsx

```diff
@@ -81,8 +82,10 @@ const DailyProvider: React.FC<{children: React.ReactNode}> = ({children}) => {
       setParticipantsSortOrder(peerId);
     };
 
-    const onError = ({error}: DailyEventObject<'error'>) => {
-      Sentry.captureException(error);
+    const onError = (errorEvent: DailyEventObject<'error'>) => {
+      //Seems we only get here when it has totally failed
+      setHasFailed();
+      Sentry.captureException(errorEvent.errorMsg);
```

Comment:
needs a `new Error` too, right?

Reply:
Nice, wasn't there before either so we might have missed these errors